doc/tutorial/tutorial.mdk

```diff
@@ -2203,8 +2203,8 @@ it is shadowed by the local `x`. The other cases are straightforward.
 
 > _Note for experts: Because we will only reduce closed expressions,
   where all variables
-  are bound by previous lambdas, we will ever only substitute closed
-  expressions (closed values in fact), and this naive definition of

```

Comment:
Ok, I think we agree on the situation, but my comment was unclear. I agree that the original text is correct, I just think mentioning that `e` is a value is a bit distracting in a discussion about capture-avoiding substitution, even for experts.

My patch removes "closed values in fact" and throws in a mention that in`EApp e1 e2` we reduce `e2` due to evaluating call-by-value, just so this is mentioned somewhere.